src/Analysis/Ast/Impl/Microsoft-Python-Analysis.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft-Python-Analysis</id>
+    <version>$version$</version>
+    <title>Microsoft Python Static Analysis</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <license type="expression">Apache-2.0</license>
+    <projectUrl>https://github.com/Microsoft/python-language-server</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Microsoft Python Static Code Analysis Engine</description>
+    <releaseNotes>Initial release.</releaseNotes>
+    <copyright>Copyright (c) 2018-2019</copyright>
+    <tags>Python</tags>
+  </metadata>
+  <files>
+    <file src="Microsoft.Python.Core.dll" target="lib/netstandard2.0/Microsoft.Python.Core.dll" />
+    <file src="Microsoft.Python.Analysis.Core.dll" target="lib/netstandard2.0/Microsoft.Python.Analysis.Core.dll" />
+    <file src="Microsoft.Python.Parsing.dll" target="lib/netstandard2.0/Microsoft.Python.Parsing.dll" />
+    <file src="Microsoft.Python.Analysis.dll" target="lib/netstandard2.0/Microsoft.Python.Analysis.dll" />
+  </files>
```

Comment:
I'm not experienced with .nuspec, but since you have a dependency on `Microsoft.Extensions.FileSystemGlobbing`, should you declare it as a dependent package here?

Reply:
Yes, added